.browserslistrc

```diff
@@ -1,3 +1,4 @@
 current node
 last 2 chrome versions
 last 2 firefox versions
+last 2 safari versions
```

Comment:
Added to get the `-webkit-` CSS prefixes from autoprefixer. We could also decide that we don't support safari (we don't _really_) but at least this line is easy to add.